app/daos/OrganizationDAO.scala

```diff
@@ -145,14 +174,29 @@ class MariadbOrganizationDAO extends OrganizationDAO {
     }yield (o, b)
     dbConfig.db.run(action.result).map(ob => {OrganizationConverter.buildOrganizationBankaccountFromResult(ob)})
   }
+  def withBankaccounts(name: String): Future[Option[Organization]] = {
+    val action = for {
+      o <- organizations.filter(o => o.name === name)
+      b <- bankaccounts.filter(b => b.organization_id === o.id)
+
+    }yield (o, b)
+    dbConfig.db.run(action.result).map(ob => {OrganizationConverter.buildOrganizationBankaccountFromResult(ob)})
+  }
+
 
   def addBankaccount(bankaccount: Bankaccount, organizationId: UUID): Future[Option[Organization]] = {
     dbConfig.db.run(organizations.filter(o => o.publicId === organizationId).result).flatMap( o => {
       dbConfig.db.run((bankaccounts returning bankaccounts.map(_.id) +=  BankaccountDB(0, bankaccount.bankName, bankaccount.number, bankaccount.blz, bankaccount.iban, bankaccount.bic, o.head.id)))
     })
     find(organizationId)
```

Comment:
I think here you have implemented a possible error. Your `dbConfig.db.run` runs in parallel to your `find` call. Thus, your find can be done before `dbConfig.db.run` has added the new bank account. Try to call a `.map(find(organizationId))`.